backend/src/database/migrations/20230221073046-create-api-keys-table.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('api_keys', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.BIGINT,
+      },
+      user_id: {
```

Comment:
probably a worthwhile effort to create index on this col now before the user size api key table grows big.

also remember there was a set active / inactive functionality briefly discussed before, i'm guessing that's no longer required!

Reply:
- The indexing piece is interesting 🤔 we might actually need cos in the future we expiring keys
- Yea the expiry, but we're not doing it here yet. This is just for multi API keys first